extensions/wikia/AdEngine/js/spec/video/playwire.spec.js

```diff
@@ -0,0 +1,83 @@
+/*global describe, it, expect, modules, beforeEach, spyOn*/
+describe('ext.wikia.adEngine.video.playwire', function () {
+	'use strict';
+
+	function noop () {}
+
+	var mocks = {
+			dfpVastUrl: {
+				build: function () {
+					return '//vast.url';
+				}
+			},
+			doc: {
+				createElement: function () {
+					return {
+						setAttribute: function (key, value) {
+							this[key] = value;
+						}
+					};
+				}
+			},
+			log: noop,
+			parent: {
+				appendChild: noop
+			}
+		};
+
+	function getModule() {
+		return modules['ext.wikia.adEngine.video.playwire'](
+			mocks.dfpVastUrl,
+			mocks.doc,
+			mocks.log
+		);
+	}
+
+	beforeEach(function () {
+		mocks.parent.appendChild = noop;
+	});
+
+	it('Get config url', function () {
+		var playwire = getModule();
+
+		expect(playwire.getConfigUrl(1, 2))
+			.toEqual('//config.playwire.com/1/videos/v2/2/zeus.json');
+	});
+
+	it('Get config url', function () {
+		var playwire = getModule();
+
+		expect(playwire.getConfigUrl(123, 789))
```

Comment:
What do you think about passing object to `getConfigUrl` to making it more understandable?